lib/matplotlib/collections.py

```diff
@@ -476,7 +480,15 @@ def set_linewidth(self, lw):
         ACCEPTS: float or sequence of floats
         """
         if lw is None:
-            lw = mpl.rcParams['patch.linewidth']
+            if (self._edge_default or
+                mpl.rcParams['_internal.classic_mode'] or
+                not self._is_filled):
```

Comment:
I am *this* close to making a PR that modifies the pep8 tests to treat the above indentation as wrong